core/src/main/java/org/springframework/security/core/context/SecurityContextHolder.java

```diff
@@ -110,6 +110,14 @@ public static SecurityContext getContext() {
 		return strategy.getContext();
 	}
 
+	/**
+	 * Peeks the current <code>SecurityContext</code>.
+	 * @return the security context (may be <code>null</code>)
```

Comment:
For each new public method that you introduce, will you please add `@since 5.6` to the JavaDoc?